lib/wovnrb/store.rb

```diff
@@ -185,14 +178,47 @@ def wovn_host
     end
 
     private
-    def cleanSettings
-      @settings['ignore_globs'] = []
-    end
 
     def stringify_keys!(h)
       h.keys.each do |k|
         h[k.to_s] = h.delete(k)
       end
     end
   end
+
+  class Settings < Hash
+    def initialize(*args, **kwargs)
+      super(*args, **kwargs)
+      @dynamic = {}
```

Comment:
`dynamic` is adjective.
why not change name to dynamic_setting or something noun?

Reply:
I renamed to `dynamic_settings`